std/math.d

```diff
@@ -6279,12 +6279,26 @@ Unqual!(Largest!(F, G)) pow(F, G)(F x, G y) @nogc @trusted pure nothrow
         {
             // Result is real only if y is an integer
             // Check for a non-zero fractional part
-            if (y > -1.0 / real.epsilon && y < 1.0 / real.epsilon)
-            {
-                long w = cast(long)y;
-                if (w != y)
+            enum maxOdd = pow(2.0L, real.mant_dig) - 1.0L;
+            static if(maxOdd > ulong.max) {
```

Comment:
Wouldn't this static if condition always be false?  The only way it could ever succeed is if CTFE started supporting ``ucent``.